pkg/scheduler/actions/reclaim/reclaim.go

```diff
@@ -106,18 +106,21 @@ func (ra *Action) Execute(ssn *framework.Session) {
 
```

Comment:
Thank you for discovering and fixing this problem. Is it possible to add `queues.Push(queue)` on line 200 of the loop here?

There is similar processing in allocate.

Reply:
I will change the order of the code.